trino/trino.go

```diff
@@ -836,6 +836,14 @@ func parseType(name string) []string {
 	return parts
 }
 
+func mapStrings(vs []string, f func(string) string) []string {
```

Comment:
I think it would be more easier to just add 
```
	name = strings.ToLower(name)
```
as a first line in `parseType` :)

Reply:
🤦 I guess that's exactly why we have code reviews! Done, thanks!